services/bookables-api/package.json

```diff
@@ -4,7 +4,7 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "test": "cd ../../ && node node_modules/.bin/jest bookables-api"
+    "test": "cd ../../ && node node_modules/.bin/jest bookables-api --coverage"
```

Comment:
Do we really need the --coverage option? The developer could add it by himself when running the script like:

`yarn test --coverage`

Reply:
fixed